### ISSUE TITLE
joplin-renderer/assets/katex/katex.css should be updated according to KaTeX 0.12.0.

### ISSUE THREAD

**Issue by barro350** [laurent22/joplin#3664]
## Environment

Joplin version: v1.0.24
Platform: Windows, but relevant to any platforms
OS specifics: 

## Steps to reproduce

1. Fresh install Joplin v1.0.234 or v1.0.235.
2. Make sure that KaTeX plugin is enabled.
3. `$a$` is rendered as an upright font, not italic.

---

## Describe what you expected to happen

As in v1.0.233, `$a$` should be properly rendered as italic.

## How to fix

Since [this commit](https://github.com/laurent22/joplin/commit/5f46d60c5e02a6f948a09e135e30c6bf33d078d2), KaTeX has been upgraded from v0.11.0 to v0.12.0.

KaTeX has changed the name of a span class `span.mathdefault` to `span.mathnormal`. For example, while KaTeX v0.11.0 renders LaTeX syntax `$a$` to HTML `<span class="mord mathdefault">a</span>`, KaTeX v0.12.0 renders `$a$` to `$<span class="mord mathnormal">a</span>$`.

Joplin uses `katex.css` in `joplin-renderer/assets/katex/katex.css` instead of the CSS file downloaded by npm to `node_modules`. Therefore, `joplin-renderer/assets/katex/katex.css` should be manually updated to <https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css>.

**Comment by tessus** [comment]
Yes, this is correct. I've noticed this as well, when testing/adding a newer Mermaid version.
I'm still waiting for Laurent's answer in the forum, otherwise I would have already commited the new assets (including the katex ones).
